cfme/utils/log.py

```diff
@@ -465,12 +465,20 @@ def _configure_warnings():
     # hack to avoid circular imports
     maybe_appliance = sys.modules.get('cfme.utils.appliance')
     if maybe_appliance is not None:
-        # TODO opt-out
-        warnings.simplefilter(
-            'ignore', maybe_appliance.NavigatableDeprecationWarning)
-
-        warnings.simplefilter('error',
-            maybe_appliance.ApplianceSummoningWarning)
+        # TODO opt-out, follow up with the location of configuring things
+        # these currently cause warnings in case something bad happens
+        # the followup will reposition the setup so it no longer incurrs the issues
+        try:
+            warnings.simplefilter(
+                'ignore', maybe_appliance.NavigatableDeprecationWarning)
+        except AttributeError as e:
+            warnings.warn("incomplete appliance module: %s" % e)
```

Comment:
**QUESTION:**  I'm not sure what that warning means.    In other words if I saw the warning "incomplete appliance module" I would have no clue as to the danger or the remedy.   Is there a better way of wording that?

Reply:
**QUESTION**: Does `warnings` use the same formatting system as logging? If yes, then the `%` is unnecessary, if not, `{}` may probably be a better option, whatchu think?

Reply:
@mfalesni warnings doesn't format, thanks for the note on the convention

@jamesooden basically if at that stage we import the appliance module and logging is not set up, we are in deep trouble, currently that trouble can happen based on import order - it will take time to restructure it so it can be safe in all cases

Reply:
I don't know how to word that better then.   I get it now.